tests/main/ubuntu-core-classic/task.yaml

```diff
@@ -0,0 +1,40 @@
+summary: Ensure classic dimension works correctly
+systems: [ubuntu-core-16-64]
+environment:
+    SUDO_USER: test
```

Comment:
Thansk, this is indeed a bit unclear as it is. I added a comment now into the code what it is used for:
```
+    # We need to set the SUDO_USER here to simulate the real
+    # behavior. I.e. when entering classic it happens via
+    # `sudo classic` and the user gets a user shell inside
+    # the classic environment that has sudo support.
```